src/otx/algo/common/layers/position_embed.py

```diff
@@ -8,9 +8,49 @@
 import math
 
 import torch
-from torch import nn
+from torch import Tensor, nn
 
-from otx.algo.object_detection_3d.utils.utils import NestedTensor
+
+class NestedTensor:
+    """Nested tensor class for object detection 3D models."""
```

Comment:
Is that tensor actually used in tasks other than 3d det? This docstring is a bit confusing